fluentd/vendored_gem_src/syslog_protocol/lib/syslog_protocol/packet.rb

```diff
@@ -7,6 +7,7 @@ class Packet
 
     def initialize
       @rfc = :rfc3164
+      @appname = 'fluentd' ##TODO(vimalk78) change it to NILVALUE
```

Comment:
does this need to be addressed now?

Reply:
This is se to value `'fluentd'` because the tests in CLO master repo checks for this.